src/Microsoft.CodeQuality.Analyzers/Core/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1343,4 +1343,7 @@
   <data name="AssigningSymbolAndItsMemberInSameStatementTitle" xml:space="preserve">
     <value>Assigning symbol and its member in the same statement.</value>
   </data>
+  <data name="DoNotIgnoreMethodResultsMessageDisposable" xml:space="preserve">
+    <value>{0} calls {1} which is never disposed. Ensure this is the intended use.</value>
```

Comment:
Please feel free to suggest a better message. On the ticket the message was more assertive but IMO we cannot guarantee that a method returning some `IDisposable` expect the caller to dispose it. The `IDisposable` interface mixes the notion of owned resource and used/shared resource.